src/Plugin/EmulateElastic/ClusterKibanaHandler.php

```diff
@@ -0,0 +1,1401 @@
+						'version_map_memory_in_bytes' => 0,
+					],
+					'shards' => [
+						'index' => [
+							'primaries' => [
+								'avg' => 1.0,
+								'max' => 1,
+							],
+							'replication' => [
+								'avg' => 0.0,
+								'max' => 0.0,
+								'min' => 0.0,
+							],
+							'shards' => [
+								'avg' => 1.0,
+								'max' => 1,
+								'min' => 1,
+							],
+						],
+						'primaries' => 3,
+						'replication' => 0.0,
+						'total' => 3,
+					],
+					'store' => [
+						'size_in_bytes' => 40113,
+					],
+				],
+				'nodes' => [
+					'count' => [
+						'coordinating_only' => 0,
+						'data' => 1,
+						'ingest' => 1,
+						'master' => 1,
+						'ml' => 0,
+						'total' => 1,
+						'voting_only' => 0,
+					],
+					'discovery_types' => [
+						'single-node' => 1,
+					],
+					'fs' => [
+						'available_in_bytes' => 93619466240,
+						'free_in_bytes' => 103636197376,
+						'total_in_bytes' => 195721834496,
+					],
+					'ingest' => [
+						'number_of_pipelines' => 1,
+						'processor_stats' => [
+							'gsub' => [
+								'count' => 0,
+								'current' => 0,
+								'failed' => 0,
+								'time_in_millis' => 0,
+							],
+							'script' => [
+								'count' => 0,
+								'current' => 0,
+								'failed' => 0,
+								'time_in_millis' => 0,
+							],
+						],
+					],
+					'jvm' => [
+						'max_uptime_in_millis' => 180864407,
+						'mem' => [
+							'heap_max_in_bytes' => 1056309248,
+							'heap_used_in_bytes' => 196453072,
+						],
+						'threads' => 44,
+						'versions' => [
+							[
+								'bundled_jdk' => true,
+								'count' => 1,
+								'using_bundled_jdk' => true,
+								'version' => '13.0.2',
+								'vm_name' => 'OpenJDK 64-Bit Server VM',
+								'vm_vendor' => 'AdoptOpenJDK',
+								'vm_version' => '13.0.2+8',
+							],
+						],
+					],
+					'network_types' => [
+						'http_types' => [
+							'security4' => 1,
+						],
+						'transport_types' => [
+							'security4' => 1,
+						],
+					],
+					'os' => [
+						'allocated_processors' => 2,
+						'available_processors' => 2,
+						'mem' => [
+							'free_in_bytes' => 804139008,
+							'free_percent' => 10,
+							'total_in_bytes' => 8315695104,
+							'used_in_bytes' => 7511556096,
+							'used_percent' => 90,
+						],
+						'names' => [
+							[
+								'count' => 1,
+								'name' => 'Linux',
+							],
+						],
+						'pretty_names' => [
+							[
+								'count' => 1,
+								'pretty_name' => 'CentOS Linux 7 (Core)',
+							],
+						],
+					],
+					'packaging_types' => [
+						[
+							'count' => 1,
+							'flavor' => 'default',
+							'type' => 'docker',
+						],
+					],
+					'plugins' => [],
+					'process' => [
+						'cpu' => [
+							'percent' => 0,
+						],
+						'open_file_descriptors' => [
+							'avg' => 232,
+							'max' => 232,
+							'min' => 232,
+						],
+					],
+					'versions' => [
+						'7.10.0',
+					],
+				],
+				'status' => 'green',
```

Comment:
Is it a type of config that we use and display sometimes? For better readability and maintenance, maybe we should consider putting it in a file and caching it once on load. This would make the file easier to read and edit in the future. It seems like mostly static config, so it's a good idea to keep it in a separate file.

Reply:
Yes, I agree.